api/server.go

```diff
@@ -94,9 +94,7 @@ func (server *Server) authorize(next http.Handler) http.Handler {
 			next.ServeHTTP(w, r)
 			return
 		}
-		if err != nil {
-			server.log.Error("failed to decode API key string", zap.Error(Error.Wrap(err)))
-		}
+
```

Comment:
Without this error check, if there is a decoding error, you will not be able to distinguish it from a failure verifying the API key. Also, the decoding error would never be logged.

Reply:
That's exactly why I removed it, since I can add whatever http header I want when sending the request and spam log file. IMHO Error level logging should be used for application errors which notifies that there is something worth with the app. However in this case it is user dependent error and app is working like it should. If api key provided with unexpected encoding that's also means request is not authorised and the rest of the code treats it this way.